main/zoe/guide/contracts/README.md

```diff
@@ -6,27 +6,79 @@ While Zoe provides the means to build custom smart contracts, there is a good ch
 want to use one that follows a commonly-used structure. Therefore, we currently provide several
 pre-built contracts that can be imported and run on Zoe:
 
+## Oracles
+
 - [Oracle Contract](./oracle.md) - the low-level oracle contract that
   can be used for querying [Chainlink
   oracles](https://docs.chain.link/docs/request-and-receive-data#config)
-  or other oracles.
+  or other oracles. 
+
+- [PriceAuthority](/zoe/guide/price-authority.md): To use an price
+  oracle in your own contract, we recommend using the `priceAuthority`
+  higher-level abstraction.
+
+## DeFi Contracts
+
+These contracts create some sort of financial instrument.
+
+
+- [Call Spread](./callSpread.md) - Creates a pair of fully collateralized call spread
+  options. They are ERTP assets and can be used as such in other contracts.
 - [Loan Contract](./loan.md) - a basic collateralized loan contract.
-- [Atomic Swap](./atomic-swap.md) - A basic trade of digital assets between two parties.
+- [Covered Call](./covered-call.md) - Creates a call option, the right
+  to buy an underlying asset.
+- [OTC Desk](./otc-desk.md) - A contract for giving quotes that can be
+  exercised. The quotes are guaranteed to be exercisable because they
+  are actually options with the underlying assets escrowed.
+
+## Uniswap-like Constant Product AMMs
+
 - [Autoswap](./autoswap.md) - An automated market maker (implementating Uniswap v1) that
   can always trade between the two currencies in its liquidity pool.
-- [Barter Exchange](./barter-exchange.md) - An exchange with an order book that allows all
-  kinds of goods to be offered for explicit barter swaps.
-- [Call Spread](./callSpread.md) - Creates a pair of fully collateralized call spread
-  options. They are ERTP assets and can be used as such in other contracts.
-- [Covered Call](./covered-call.md) - This type of contract oversees the sale and execution of a
-  call option.
 - [Multipool Autoswap](./multipoolAutoswap.md) - An automated market maker implementing
   Uniswap v1. It has multiple liquidity pools and can trade between any pair of funded
   currencies.
+
+## Generic Sales/Trading Contracts
+
+These contracts involve the trading or selling of ERTP digital assets.
+Some contracts are restricted to non-fungible tokens, but most can be
```

Comment:
The only one even mentioning nft is Sell Items, and it doesn't say it's restricted to them. I suggest noting for each if it's restricted to nft or if it can use ft as well. If only Sell Items is restricted, I'd remove this and just note its restriction in the sell items entry.

Reply:
Ok, I will just remove this